Yoast/wordpress-seo
# Output escaping partial alerts template

## Summary

This PR can be summarized in the following changelog entry:

* Improves output escaping.

## Relevant technical choices:

* Added escaping to the notification's render method. This could be changed to escaping in the template, but this is more precise.
* Added `phpcs:ignore` comments for already escaped variables.
* Changed the attribute key escaping for the form fieldset and input select classes. Now using `sanitize_key`, see: https://github.com/WordPress/WordPress-Coding-Standards/pull/1786

## Test instructions

This PR can be tested by following these steps:

* Run `composer before-premium-cs`. This installs the latest Yoast CS.
* Run the `WordPress.Security.EscapeOutput` sniff on the touched files:
```
vendor/bin/phpcs -p --standard=WordPress --sniffs=WordPress.Security.EscapeOutput --report=full,summary,source --basepath=./ admin/views/partial-alerts-template.php
```
* Run `composer after-premium-cs`. This reverts the PHP packages back to normal.
* Check if the notifications still have all the HTML that they had before.

## UI changes

## Documentation

## Quality assurance


Fixes #
